src/sentry/runner/commands/cleanup.py

```diff
@@ -154,6 +155,35 @@ def cleanup(
     this can be done with the `--project` or `--organization` flags respectively,
     which accepts a project/organization ID or a string with the form `org/project` where both are slugs.
     """
+    # Tests already configure the app before calling _cleanup(),
+    # thus, we do it here to avoid conflicts when running tests
+    from sentry.runner import configure
+
+    configure()
+
+    _cleanup(
+        model=model,
+        days=days,
+        project=project,
+        organization=organization,
+        concurrency=concurrency,
+        silent=silent,
+        router=router,
+        timed=timed,
+    )
+
+
+def _cleanup(
+    model: tuple[str, ...],
+    days: int = 30,
+    project: str | None = None,
+    organization: str | None = None,
+    concurrency: int = 1,
+    silent: bool = False,
+    router: str | None = None,
+    timed: bool = False,
+    disable_multiprocessing: bool = False,
```

Comment:
New parameter only used for tests.

The processes were not able to see the groups I created during the test initialization.